account-registration/src/main/java/com/amazon/aws/spinnaker/plugin/registration/AccountsStatus.java

```diff
@@ -159,11 +171,11 @@ private void buildDesiredAccountConfig(HashMap<String, CredentialsConfig.Account
         }
         for (String ecsAccountsToRemove : accountsToCheck) {
             String ecsAccountName = ecsAccountsToRemove + "-ecs";
-            log.debug("ECS account, {}, will be removed.", ecsAccountName);
+            log.info("ECS account, {}, will be removed.", ecsAccountName);
             ecsAccounts.remove(ecsAccountName);
         }
-        log.debug("Accounts to be updated: {}", ec2Accounts);
-        log.debug("ECS accounts to be updated: {}", ecsAccounts);
+        log.debug("Accounts to be updated in CredentialsConfig: {}", ec2Accounts.keySet());
+        log.debug("ECS accounts to be updated in CredentialsConfig: {}", ecsAccounts.keySet());
```

Comment:
One thing to note about the above log lines is that depending on the number of accounts this can result in a big dump of account info.

Reply:
alternatively you can do `log.info` for the count of accounts and `log.debug` for the `keySet()`